README.md

```diff
@@ -78,16 +75,14 @@ In this example `TEST_ENV="exists"` gets passed into the container.
 <!-- Sample from ./dist/index.html -->
 
 <p>Replace Env Var: ${TEST_ENV}</p>
```

Comment:
From reading the docs, I'm pretty sure this will work, which will help alleviate confusion between environment variable and template literal interpolation. 

```suggestion
<p>Replace Env Var: $TEST_ENV</p>
```

Reply:
It's more of an example (probably not a good one) of how to target only environmental variables in the event you're in a file where this is an issue. I'll adjust the example.